todo_app/templates/index.html

```diff
@@ -12,7 +12,15 @@ <h1 class="display-4">To-Do App</h1>
       <h2>Items</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
+        {% for item in items %}
+          <li>{{ item.title }} - {{item.status}}</li>
+        {% endfor %}
       </ul>
+        <form action = "/add_item" method = "POST">
+          <label for="Add New Item">Add New Item: </label>
+          <input id="title" type="text" name="title">
```

Comment:
There are some attributes you could add, e.g. `required` so that user can't submit a blank item.